docs/ReferenceGuides/UnderscoredAttributes.md

```diff
@@ -296,10 +296,40 @@ func baz() {
 Additionally, if they are of a tuple or struct type, their stored members
 without observers may also be passed inout as non-ephemeral pointers.
 
-For more details, see [the educational note on temporary pointer usage](/userdocs/diagnostics/temporary-pointers.md).
+For more details, see the educational note on
+[temporary pointer usage](/userdocs/diagnostics/temporary-pointers.md).
 
 ## `@_nonoverride`
 
+Warns when a protocol restates a requirement from another protocol that
```

Comment:
How about, "Marks a declaration that is not an override of another, required when compiling with the `-warn-implicit-overrides` flag, which warns when a protocol restates a requirement from another protocol that it refines without annotating the declaration with either `override` or `@_nonoverride`."

Reply:
Updated wording.